### DISCUSSION TITLE
Sprite sheet export

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#665] [Ideas]
Allow us to export ToonSquid animations to a sprite sheet for games.

**Comment by freeaces** [comment]
I would really love sprite sheet import and export. Would be a big help for making games.

**Comment by DWrightAnimation** [comment]
If I ended up learning how to make games, or maybe create a series that someone would want to work with me to make into a game, this would be the most affordable way to create the sprites for those games.

**Comment by Lioncub901** [comment]
I remembered I wrote this

**Comment by overcesium** [comment]
This

**Reply by shimonshomron** [reply]
+1 from web designer perspective. super useful.
